controllers/suite_test.go

```diff
@@ -30,7 +30,6 @@ import (
 
 	nifiv1 "github.com/konpyutaika/nifikop/api/v1"
 	nifiv1alpha1 "github.com/konpyutaika/nifikop/api/v1alpha1"
-	//+kubebuilder:scaffold:imports
```

Comment:
i'm not sure if this will cause problems. Was this done by the formatter?

Reply:
Good catch. I will add the exclusions for the project.